dev/NavigationView/NavigationViewItem.h

```diff
@@ -63,6 +63,8 @@ class NavigationViewItem :
     bool IsRepeaterVisible() const;
     void PropagateDepthToChildren(int depth);
     bool HasChildren();
+    // Needed for scenarios where the Repeater is not loaded (OnApplyTemplate) therefore we cannot guarantee a non-null MenuItemsSource acutally contains items
```

Comment:
ItemsRepeater.
(I would love to replace Repeater with ItemsRepeater everywhere. It just bothers me :-))